package.json

```diff
@@ -2,23 +2,24 @@
   "name": "handsontable-multi-select",
   "version": "0.27.9",
   "description": "Handsontable editor and renderer for multi-select cells",
-  "main": "dist/js/handsontable-multi-select.js",
+  "main": "dist/multi-select.js",
```

Comment:
this should not be changed, it should still be `dist/js/handsontable-multi-select.js`